gtk4/src/auto/drag_icon.rs

```diff
@@ -20,6 +20,15 @@ glib::wrapper! {
 }
 
 impl DragIcon {
+    #[doc(alias = "gtk_drag_icon_get_for_drag")]
+    pub fn get_for_drag(drag: &gdk::Drag) -> DragIcon {
```

Comment:
This one is odd.

Reply:
no, that is expected, it wasn't marked as a constructor

Reply:
I see it as ctor in the new gir.

Reply:
The get prefix is kind of wrong, should be manually renamed I guess

Reply:
Done.